README.md

```diff
@@ -529,6 +530,7 @@ Fields listed under `task_definition` correspond to fields that will be included
   * If you are using Docker compose version 3, the `cpu_shares`, `mem_limit`, and `mem_reservation` fields are optional and must be specified in the ECS params file rather than the compose file.
   * In Docker compose version 2, the `cpu_shares`, `mem_limit`, and `mem_reservation` fields can be specified in either the compose or ECS params file. If they are specified in the ECS params file, the values will override values present in the compose file.
   * If you are using a private repository for pulling images, `repository_credentials` allows you to specify an AWS Secrets Manager secret ARN for the name of the secret containing your private repository credentials as a `credential_parameter`.
+  * `gpu` is an optional parameter that maps to the [GPU resource requirement](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ResourceRequirement.html) for your container.
```

Comment:
I think it'd be good to include an example of valid values.

Reply:
Might be worth specifying a full example with the GPU instance type as well.

Reply:
I suggest being more descriptive re: the what the value actually is:

"`gpu` is the number of physical GPUs the Amazon ECS container agent will reserve for the container. Maps to the GPU [resource requirement](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ResourceRequirement.html) field in the task definition."

Reply:
@SoManyHs added examples for `gpu`. I'm not sure if we need to do anything for the instance type since that's just an additional value for the `--instance-type` argument.

@allisaurus done!

Reply:
Added also a description for `init_process_enabled`.